Cargo.lock

```diff
@@ -21,6 +21,15 @@ dependencies = [
  "memchr",
 ]
 
+[[package]]
+name = "ansi_term"
+version = "0.11.0"
```

Comment:
Similarly, this pulls in a second (older) version of `ansi_term`. It would be best if we could avoid these duplicate deps.